src/components/Button.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+interface ButtonProps {
+  label: React.ReactNode;
+  onClick?: () => void;
+  className?: string;
+  type: "button" | "submit";
+}
+
+const Button: React.FC<ButtonProps> = ({
```

Comment:
Please check the reusability of the button component and add variants to it to make it easier to reuse. For example, if the button has different colors, add corresponding variants for each color.